pkg/pillar/containerd/oci.go

```diff
@@ -245,6 +245,18 @@ func (s *ociSpec) Load(file *os.File) error {
 	if s.Annotations == nil {
 		s.Annotations = map[string]string{}
 	}
+	// default OCI specs have all devices being denied by default,
```

Comment:
As you explained, this code is already part of the [NewOciSpec()](https://github.com/lf-edge/eve/blob/6df1b5396d2c5132fef54b6e8a76c8caee123121/pkg/pillar/containerd/oci.go#L70), I think is worth the put a comment here why are you adding to the loader, i.e., to ensure that containers with old OCI spec will also have the correct resource allocation performed...

Reply:
Yes, exactly, thank you for adding this comment @rene!